### ISSUE TITLE
Add a 'feedback' link on every page

### ISSUE THREAD

**Issue by matthiku** [matthiku/cSpot#71]
In order for users to quickly give feedback (report issues or request enhancements), the UI should provide a quick link to a feedback-popup on every page. 

The feedback should be sent to the email address of users with the role 'Administrator'.

**Comment by matthiku** [comment]
1. Add link and popup-modal to the main navbar
2. Create a feedback controller to handle the feedback mailing and storage

Q.: Should there be a local storage for feedback?

**Comment by matthiku** [comment]
Resolved by implementing https://github.com/cmgmyr/laravel-messenger

**Comment by matthiku** [comment]
Still giving an error when trying to send a message:
> 
ErrorException in Connection.php line 669: 
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'users.name' in 'field list' (SQL: select concat(users.name) as name from `users` inner join `participants` on `users`.`id` = `participants`.`user_id` where `participants`.`thread_id` = 1 and `users`.`id` != 1) (View: C:\wamp\www\dev\cSpot\resources\views\messenger\index.blade.php)

**Comment by matthiku** [comment]
This is caused by the messenger plugin which tries to find a 'name' field in the users table. I could fix it in the plugin, but then it would always be broken again when the plugin gets updated.

Instead, in the 'users' table, I added the 'name' field back in order to avoid any future issues like this.

Also, in the User model, I added this Mutator:
public function setFirstNameAttribute($first_name)
```
{
public function setFirstNameAttribute($first_name)
    $this->attributes['first_name'] = $first_name;
    $this->attributes['name'] = $first_name;
}
```
